base/applications/regedit/main.h

```diff
@@ -134,6 +134,7 @@ BOOL IsDefaultValue(HWND hwndLV, int i);
 /* regedit.c */
 void WINAPIV output_message(unsigned int id, ...);
 void WINAPIV error_exit(unsigned int id, ...);
+void WINAPIV error_dont_exit(unsigned int id, ...);
```

Comment:
I would propose to rename this function from error_dont_exit() to just error(), as I do think it makes sense to name a function with what it does, and not like what it doesn't do.
What is the difference between output_message() and error_dont_exit()? If the first one would already do the job, why not using that instead?

Reply:
Used `output_message` instead of `error_dont_exit`.
`error_dont_exit` is deleted.

Reply:
Let's hope the existing calls to that won't spam us with undesired MessageBoxes too much then. But I am fine to give it a try definitely. Resolved.